src/myr/Myr.js

```diff
@@ -421,7 +428,20 @@ class Myr {
         return this.cursor.color;
     }
 
-    getRandomColor = (colors = null) => {
+    setTexture = (texture = "blank") => {
+        for (let i=0; i < TexturePack.length; i++) {
+            if (TexturePack[i].title === texture) {
+                this.cursor.texture = TexturePack[i].url;
+            }
+        }
```

Comment:
There are much better ways of doing this. In JS, you should almost never use a for loop with an array. Look up how to use indexOf. It's also probably best to return as soon as you set the texture as you're doing extra work you don't have to.